docs/source/aws-issues.rst

```diff
@@ -61,3 +61,14 @@ resources are shutdown. These particular resources do not incur in additional
 charges, but their existance counts towards your account's quota.
 We recommend you retry the ``elastic-blast delete`` command ~24 hours later to fully
 shutdown the aforementioned resources.
+
+.. _blast_tuner_requires_csp_target:
+
+blast-tuner application requires csp-target
+-------------------------------------------
+
+If the ``--csp-target`` parameter is not specified to ``blast-tuner``, an
```

Comment:
AWS will work fine without `--csp-target`, because it defaults to AWS.

Reply:
OK, thanks. Fixed in eb29ddd